Add the main page, load discord and github logos.

Vooxify opened the pull request:
### Main Page is created
I've created main page for the new users. On the page, the login page is already accessible.
Finish:
    - **The responsive**
    - **Add the redirection to EDP Unblock**
    - **Add the 2FA security**

saumon-brule commented on src/components/Mainpage/MainPage.css:
```diff
@@ -0,0 +1,200 @@
+
+}
+
+.navbar {
```
You should remove all these empty selectors

saumon-brule commented:
- Update the theme and colors to make them match with the /login page (same background color and same color of "window")
- You replaced the EDP Logo with a "retour" button. I think that it would be understandable if the EDP Logo redirected to /main-page and I think the button takes too much space and is too visible for the login page. If you really think that click on the logo is not a good idea to redirect to the main-page, consider changing at least the style of the button (for instance the goBackArrow in the feedback page).
- Try (I don't know if you can) to change the branch on which you're about to merge with the pull request, instead (if you can) make the pull request on a new branch.

Otherwise thank you for your good proposition 👍

saumon-brule commented on src/components/Mainpage/MainPage.jsx:
```diff
@@ -0,0 +1,69 @@
+        setIsMenuOpen(!isMenuOpen);
+    };
+
+    // Fonction pour définir le lien actif
```
This function is useless 😅. You're creating a function with a parameter that is triggering another function with the same parameter. You just could use the other function instead ("setActiveLink" in this case")

Vooxify commented:
I changed what was requested, I changed the colors and the logo back. It is now the EDP logo. With my level in git, i join forked repo [ https://github.com/Magic-Fishes/Ecole-Directe-Plus/compare/main...Vooxify:Ecole-Directe-Plus:save ]. Tell me what I should do

Vooxify commented:
So, with the recent commits, I couldn't integrate the extension page. I'll let you do it.
